SCClassLibrary/Platform/Platform.sc

```diff
@@ -172,6 +172,10 @@ Platform {
 	// hook for clients to write frontend.css
 	writeClientCSS {}
 
+	killProcessByID { |cmdLineArgs|
+		^this.subclassResponsibility(\kill)
```

Comment:
this should be `^this.subclassResponsibility(\killProcessByID)`, no?

Reply:
yes right, thanks for spotting it!